### PR TITLE
test(combineAll): add combineAll tests to mirror combineLatest

### PR DESCRIPTION
**Pull request by staltz**

### CODE REVIEW

File: spec/operators/combineAll-spec.js [hunk 1]
```diff
@@ -4,6 +4,416 @@ var Observable = Rx.Observable;
...
+    };
+    var e1 =        hot('-a-^-|', values);
+    var e1subs =           '^ !';
+    var e2 =        hot('-b-^-c-|', values);
+    var e2subs =           '^   !';
+    var expected =         '----|';
+
+    var result = Observable.of(e1, e2).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, values);
```
Comment (kwonoj) [inline spec/operators/combineAll-spec.js]: is `values` required in this test case? `expected` is empty observable.

Comment (staltz) [inline spec/operators/combineAll-spec.js]: I can remove it...

**Comment by staltz** [comment]
Updated @kwonoj

File: spec/operators/combineAll-spec.js [hunk 2]
```diff
@@ -4,6 +4,416 @@ var Observable = Rx.Observable;
...
+
+    var result = Observable.of(left, right).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, null, 'bad things');
+    expectSubscriptions(left.subscriptions).toBe(leftSubs);
+    expectSubscriptions(right.subscriptions).toBe(rightSubs);
+  });
+
+  it('should handle interleaved with tail', function () {
+    var e1 = hot('-a--^--b---c---|', { a: 'a', b: 'b', c: 'c' });
```
Comment (kwonoj) [inline spec/operators/combineAll-spec.js]: (as same as line 135) does specifying same value required for this test case?

Comment (staltz) [inline spec/operators/combineAll-spec.js]: Good catch. Since these tests are from combineLatest, I'll have to update those to.

**Comment by staltz** [comment]
Updated PR @kwonoj

**Comment by kwonoj** [comment]
Cool, merged with 7fcc98e. Appreciate, @staltz :)

**Pull request closed without merging**